xdem/coreg.py

```diff
@@ -1506,7 +1534,6 @@ def apply_matrix(
     :param invert: Invert the transformation matrix.
     :param centroid: The X/Y/Z transformation centroid. Irrelevant for pure translations. Defaults to the midpoint (Z=0)
     :param resampling: The resampling method to use. Can be `nearest`, `bilinear`, `cubic` or an integer from 0-5.
-    :param dilate_mask: DEPRECATED - This option does not do anything anymore. Will be removed in the future.
```

Comment:
Wait sorry I missed something here. Why is this not needed any more?

Reply:
Because the code that was using it was deleted in a previous PR (and it was on TODO list to take the time to fully remove it).
See the issue about it https://github.com/GlacioHack/xdem/issues/315.